src/pages/components/leadspace.mdx

```diff
@@ -41,24 +56,27 @@ This option has 4 empty columns on the left and right to accommodate pages that
 
 ### Lead space left-aligned
 
-| Field                    | Required                                                                                              | Max character size (English / translated) |
-| ------------------------ | ----------------------------------------------------------------------------------------------------- | ----------------------------------------- |
-| Headline                 | Yes                                                                                                   | XS (65 / 85)                              |
-| Short copy               | No                                                                                                    | M (120 / 150)                             |
-| Secondary call to action | No                                                                                                    | M (120 / 150)                             |
-| Primary call to action   | No                                                                                                    | XXS                                       |
-| Background image         | Yes<br/> <br/><div style="max-width: 200px"> Note: each breakpoint requires a different image.</div> |                                           |
+| Field          | Field type      | Required                              | Max character size <br/>(English / translated) |  Notes          |
+| -------------- | --------------- | ------------------------------------- | ---------------------------------------------- | --------------- |
+| Headline       | Short copy      | Yes                                   | XS (65 / 85)                                   | –               |
+| Description    | Short copy      | No                                    | M (120 / 150)                                  | –               |
+| [Button group](https://www.ibm.com/standards/web/carbon-for-ibm-dotcom/components/button-group) | Component | No | XXS (40 / 55) | –        |
```

Comment:
Maybe mention button groups will not be available on `Short`? I think we decided this. Lemme know if we didn't and I was dreaming.